examples/angularjs_require/js/directives/todoEscape.js

```diff
@@ -0,0 +1,16 @@
+/*global define*/
+define(['app'], function (app) {
+	'use strict';
+
+	app.directive('todoEscape', function () {
+		var ESCAPE_KEY = 27;
+
+		return function (scope, elem, attrs) {
+			elem.bind('keydown', function (event) {
+				if (event.keyCode === ESCAPE_KEY) {
```

Comment:
obviously there is a bunch of copypasta going on here :) but I would rather not introduce something broken right?